lib/marked.js

```diff
@@ -1345,25 +1345,36 @@ marked.setOptions = function(opt) {
   return marked;
 };
 
-marked.defaults = {
-  baseUrl: null,
-  breaks: false,
-  gfm: true,
-  headerIds: true,
-  headerPrefix: '',
-  highlight: null,
-  langPrefix: 'lang-',
-  mangle: true,
-  pedantic: false,
-  renderer: new Renderer(),
-  sanitize: false,
-  sanitizer: null,
-  silent: false,
-  smartLists: false,
-  smartypants: false,
-  tables: true,
-  xhtml: false
-};
+marked.getDefaults = function () {
+  var defaults = {
+    baseUrl: null,
+    breaks: false,
+    gfm: true,
+    headerIds: true,
+    headerPrefix: '',
+    highlight: null,
+    langPrefix: 'lang-',
+    mangle: true,
+    pedantic: false,
+    renderer: new Renderer(),
+    sanitize: false,
+    sanitizer: null,
+    silent: false,
+    smartLists: false,
+    smartypants: false,
+    tables: true,
+    xhtml: false
+  };
+
+  var clone = {};
```

Comment:
Is it possible to do this as part of the test itself instead of the lib? Believe @styfle mentioned something like that before. Thinking it would be better to have the test use a blank object over the singleton pattern that seems preferred by Node (understandably in many use cases).